react-accordion-component/my-app/src/Accordian.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react"
+import './Accordian.css'
+
+export default function Accordion() {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const handlePanelClick = (index) => {
+    setActiveIndex((prevIndex) => (prevIndex === index ? 0 : index));
+  };
+
+  return (
+    <>
```

Comment:
let's try this using the map array method. We want to stay away from hard coding these <Panel> components